packages/govuk-frontend-review/src/views/component-preview.njk

```diff
@@ -1,5 +1,11 @@
 {% extends "layouts/" + previewLayout | default('layout') + ".njk" %}
 
+{% block pageTitle %}
+  {{- componentName | unslugify -}}
+  {{ ", " + exampleName | unslugify | lower if exampleName }}
+  - GOV.UK Frontend
+{%- endblock %}
+
```

Comment:
Not looked too closely but might have a couple of minor conflicts with https://github.com/alphagov/govuk-frontend/pull/4416 in this PR

Similar to https://github.com/alphagov/govuk-frontend/pull/4416/commits/c7d87769e24e725a820b00b40407867cadab37c8 probably watch out for new lines creeping in here?

```suggestion
{% set pageTitle = (componentName + ", " + exampleName | lower if exampleName else componentName) | unslugify %}
{% block pageTitle %}{{ pageTitle }} - GOV.UK Frontend{% endblock %}
```